coala_quickstart/generation/FileGlobs.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from coala_quickstart.generation.Utilities import comma_split
+from coala_quickstart.interaction.Question import ask_question
+from coala_quickstart.Strings import glob_help
+from coalib.collecting.Collectors import collect_files
+
+
+def get_file_glob_list(printer, prompt, default="**"):
```

Comment:
maybe you can generalize this function into some `ask_for_globs` function, this is generally useful imo :)